maintainers/maintainer-list.nix

```
@@ -5312,6 +5312,12 @@
     githubId = 39689;
     name = "Hugo Tavares Reis";
   };
+  hufman = {
+￼   email = "hufman@gmail.com";
+￼   github = "hufman";
+￼   githubId = 1592375;
+￼   name = "Walter Huf";
+￼ };
```

Something went horribly wrong here :^)
```suggestion
    email = "hufman@gmail.com";
    github = "hufman";
    githubId = 1592375;
    name = "Walter Huf";
  };
```

---

Bah, copy-pasted from the wrong GitHub text field, it seems!